# Japanese translations for cddagl.
# Copyright (C) 2019 ORGANIZATION
# This file is distributed under the same license as the cddagl project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
#
msgid ""
msgstr ""
"Project-Id-Version: cddagl 1.3.11.1\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2019-06-11 13:41-0400\n"
"PO-Revision-Date: 2019-05-21 05:50+0900\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language: ja\n"
"Language-Team: ja <LL@li.org>\n"
"Plural-Forms: nplurals=1; plural=0\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.6.0\n"

#: cddagl/launcher.py:139
msgid "CDDA Game Launcher started: {version}"
msgstr ""

#: cddagl/launcher.py:148
msgid ""
"Global error:\n"
"Launcher version: {version}\n"
"Type: {extype}\n"
"Value: {value}\n"
"Traceback:\n"
"{traceback}"
msgstr ""

#: cddagl/ui.py:176
msgid "64-bit"
msgstr ""

#: cddagl/ui.py:178
msgid "32-bit"
msgstr ""

#: cddagl/ui.py:182
msgid "B"
msgstr ""

#: cddagl/ui.py:183
msgid "Ki"
msgstr ""

#: cddagl/ui.py:183
msgid "Mi"
msgstr ""

#: cddagl/ui.py:183
msgid "Gi"
msgstr ""

#: cddagl/ui.py:183
msgid "Ti"
msgstr ""

#: cddagl/ui.py:183
msgid "Pi"
msgstr ""

#: cddagl/ui.py:183
msgid "Ei"
msgstr ""

#: cddagl/ui.py:184
msgid "Zi"
msgstr ""

#: cddagl/ui.py:186
#, python-format
msgid "%3.1f %s%s"
msgstr ""

#: cddagl/ui.py:188
#, python-format
msgid "%.1f %s%s"
msgstr ""

#: cddagl/ui.py:188
msgid "Yi"
msgstr ""

#: cddagl/ui.py:254
msgid "Cannot rename file"
msgstr "ファイルをリネームできません"

#: cddagl/ui.py:260
msgid ""
"\n"
"<p>The launcher failed to rename the following file: {src} to {dst}</p>\n"
"<p>When trying to rename or access {filename}, the launcher raised the\n"
"following error: {error}</p>\n"
msgstr ""
"\n"
"<p>以下のファイルのリネームに失敗しました: {src} to {dst}</p>\n"
"<p>{filename}へのアクセスまたはリネームを試みましたがエラーが発生しました\n"
": {error}</p>\n"

#: cddagl/ui.py:271
msgid ""
"\n"
"<p>No process seems to be using that file.</p>\n"
msgstr ""
"\n"
"<p>指定されたファイルを使用中のプロセスはありません。</p>\n"

#: cddagl/ui.py:275
msgid ""
"\n"
"<p>The process <strong>{image_file_name} ({pid})</strong> is currently "
"using\n"
"that file. You might need to end it if you want to retry.</p>\n"
msgstr ""
"\n"
"<p>プロセス <strong>{image_file_name} ({pid})</strong> が、指定された\n"
"ファイルを使用中です。上記プロセス終了後に再試行して下さい。</p>\n"

#: cddagl/ui.py:281
msgid "Do you want to retry renaming this file?"
msgstr "ファイルのリネームを再試行しますか？"

#: cddagl/ui.py:283
msgid "Retry renaming the file"
msgstr "ファイルのリネームを再試行"

#: cddagl/ui.py:285 cddagl/ui.py:8278 cddagl/ui.py:8340
msgid "Cancel the operation"
msgstr "中止"

#: cddagl/ui.py:324 cddagl/ui.py:347
msgid "&File"
msgstr "ファイル"

#: cddagl/ui.py:325 cddagl/ui.py:351
msgid "E&xit"
msgstr "終了"

#: cddagl/ui.py:326 cddagl/ui.py:355
msgid "&Help"
msgstr "ヘルプ"

#: cddagl/ui.py:328 cddagl/ui.py:360
msgid "&Check for update"
msgstr "アップデート確認"

#: cddagl/ui.py:329 cddagl/ui.py:366
msgid "&About CDDA Game Launcher"
msgstr "&CDDA Game Launcher について"

#: cddagl/ui.py:339 cddagl/ui.py:1398 cddagl/ui.py:3296 cddagl/ui.py:3543
msgid "Ready"
msgstr "準備完了"

#: cddagl/ui.py:419
msgid ""
"Could not find launcher latest release [HTTP {status_code}] ({reason}) "
"when requesting {url}"
msgstr ""

#: cddagl/ui.py:501 cddagl/ui.py:3776
msgid "Do not check for new version of the CDDA Game Launcher on launch"
msgstr "起動時に新しいバージョンをチェックしない"

#: cddagl/ui.py:512
msgid "Launcher update"
msgstr "アップデート"

#: cddagl/ui.py:513
msgid ""
"You are using version {version} but there is a new update for CDDA Game "
"Launcher. Would you like to update?"
msgstr "使用中の {version} よりも新しいバージョンがあります。アップデートしますか？"

#: cddagl/ui.py:518
msgid "Update the launcher"
msgstr "アップデートする"

#: cddagl/ui.py:520
msgid "Not right now"
msgstr "後で"

#: cddagl/ui.py:551
msgid "Up to date"
msgstr "最新版"

#: cddagl/ui.py:552
msgid "The CDDA Game Launcher is up to date."
msgstr "お使いの CDDA Game Launcher は最新版です。"

#: cddagl/ui.py:659 cddagl/ui.py:677
msgid "Main"
msgstr "メイン"

#: cddagl/ui.py:660 cddagl/ui.py:682
msgid "Backups"
msgstr "バックアップ"

#: cddagl/ui.py:661 cddagl/ui.py:687
msgid "Mods"
msgstr "Mod"

#: cddagl/ui.py:663 cddagl/ui.py:697
msgid "Soundpacks"
msgstr "サウンドパック"

#: cddagl/ui.py:665 cddagl/ui.py:707
msgid "Settings"
msgstr "設定"

#: cddagl/ui.py:692
msgid "Tilesets"
msgstr "タイルセット"

#: cddagl/ui.py:702
msgid "Fonts"
msgstr "フォント"

#: cddagl/ui.py:845 cddagl/ui.py:855 cddagl/ui.py:1317 cddagl/ui.py:1318
#: cddagl/ui.py:1381 cddagl/ui.py:1438 cddagl/ui.py:1574 cddagl/ui.py:1664
#: cddagl/ui.py:1732 cddagl/ui.py:1947 cddagl/ui.py:3316 cddagl/ui.py:3416
#: cddagl/ui.py:5319 cddagl/ui.py:5355 cddagl/ui.py:7750 cddagl/ui.py:7792
#: cddagl/ui.py:7829 cddagl/ui.py:7865
msgid "Unknown"
msgstr "不明"

#: cddagl/ui.py:892
msgid "Directory:"
msgstr "ディレクトリ:"

#: cddagl/ui.py:893 cddagl/ui.py:6966
msgid "Version:"
msgstr "バージョン:"

#: cddagl/ui.py:894
msgid "Build:"
msgstr "ビルド:"

#: cddagl/ui.py:895
msgid "Saves:"
msgstr "セーブ:"

#: cddagl/ui.py:897
msgid ""
"Your save directory might be large enough to cause significant delays "
"during the update process.\n"
"You might want to enable the \"Do not copy or move the save directory\" "
"option in the settings tab."
msgstr ""
"アップデートプロセス中の重要な遅延のため、セーブディレクトリは十分な領域を必要とします。\n"
"設定タブから \"セーブディレクトリをコピー・移動しない\" 設定ができます。"

#: cddagl/ui.py:901 cddagl/ui.py:1190 cddagl/ui.py:1530
msgid "Launch game"
msgstr "ゲーム起動"

#: cddagl/ui.py:902
msgid "Restore previous version"
msgstr "以前のバージョンを復元する"

#: cddagl/ui.py:903
msgid "Game"
msgstr "ゲーム"

#: cddagl/ui.py:1050 cddagl/ui.py:1199 cddagl/ui.py:1539 cddagl/ui.py:6220
msgid "auto"
msgstr "自動"

#: cddagl/ui.py:1051
msgid "before_launch"
msgstr "起動前"

#: cddagl/ui.py:1063
msgid "Game executable not found"
msgstr "実行ファイルがありません"

#: cddagl/ui.py:1085
msgid "Could not launch the game executable"
msgstr "実行ファイルの起動に失敗しました"

#: cddagl/ui.py:1088
msgid "Cannot launch game"
msgstr "起動出来ませんでした"

#: cddagl/ui.py:1090
msgid ""
"\n"
"<p>The launcher failed to start the game executable in "
"<strong>{filename}</strong> .</p>\n"
"<p>It received the following error from the operating system: {error}</p>"
"\n"
"<p>Poor antivirus products are known to detect the game binary as a "
"threat and\n"
"block its execution. A simple workaround is to add the game binary in "
"your\n"
"antivirus whitelist or select the action to trust this binary when "
"detected.</p>\n"
msgstr ""
"\n"
"<p>ゲーム実行ファイルの起動に失敗しました。:<strong>{filename}</strong></p>\n"
"<p>システムから以下のエラーを受け取りました。: {error}</p>\n"
"<p>一部のアンチウイルス製品はゲームバイナリを脅威と誤認し、その起動をブロックしてし\n"
"まうことがあります。ゲームバイナリをホワイトリストに登録するか、ブロックされた際に「信頼す\n"
"る」を選択することで回避できるかもしれません。\n"

#: cddagl/ui.py:1097
msgid "[unknown]"
msgstr "[不明]"

#: cddagl/ui.py:1101 cddagl/ui.py:2741 cddagl/ui.py:3609
msgid "OK"
msgstr ""

#: cddagl/ui.py:1116 cddagl/ui.py:1401 cddagl/ui.py:1478 cddagl/ui.py:3301
#: cddagl/ui.py:3546
msgid "Game process is running"
msgstr "ゲーム実行中"

#: cddagl/ui.py:1134 cddagl/ui.py:1496
msgid "Show current game"
msgstr "現在のゲームを表示"

#: cddagl/ui.py:1172 cddagl/ui.py:1520
msgid "Game process has ended"
msgstr "ゲーム実行終了"

#: cddagl/ui.py:1200 cddagl/ui.py:1540
msgid "after_end"
msgstr "終了後"

#: cddagl/ui.py:1258
msgid "Game directory"
msgstr "ゲームディレクトリ"

#: cddagl/ui.py:1280
msgid "Not a valid directory"
msgstr "無効なディレクトリ"

#: cddagl/ui.py:1293 cddagl/ui.py:1656
msgid "console"
msgstr "コンソール"

#: cddagl/ui.py:1296 cddagl/ui.py:1659
msgid "tiles"
msgstr "タイル"

#: cddagl/ui.py:1300 cddagl/ui.py:1663
msgid "Not a CDDA directory"
msgstr "CDDAディレクトリではありません"

#: cddagl/ui.py:1313 cddagl/ui.py:3179 cddagl/ui.py:3462
msgid "Install game"
msgstr "インストール"

#: cddagl/ui.py:1324 cddagl/ui.py:1996 cddagl/ui.py:3177 cddagl/ui.py:3445
msgid "Update game"
msgstr "アップデート"

#: cddagl/ui.py:1351 cddagl/ui.py:1702
msgid "Reading: {0}"
msgstr "読み込み中: {0}"

#: cddagl/ui.py:1386 cddagl/ui.py:1734
msgid "{version} ({type})"
msgstr ""

#: cddagl/ui.py:1396
msgid "Previous version restored"
msgstr "以前のバージョンが復元されました"

#: cddagl/ui.py:1413 cddagl/ui.py:1741
msgid "{build} ({time_delta})"
msgstr ""

#: cddagl/ui.py:1431 cddagl/ui.py:3149 cddagl/ui.py:3457
msgid "Your game is up to date"
msgstr "ゲームは最新版です"

#: cddagl/ui.py:1433 cddagl/ui.py:3151 cddagl/ui.py:3459
msgid "There is a new update available"
msgstr "アップデート可能です"

#: cddagl/ui.py:1578
msgid "Not found"
msgstr ""

#: cddagl/ui.py:1613
msgid "World"
msgid_plural "Worlds"
msgstr[0] "ワールド"

#: cddagl/ui.py:1615
msgid "Character"
msgid_plural "Characters"
msgstr[0] "キャラクター"

#: cddagl/ui.py:1618
msgid "{world_count} {worlds} - {character_count} {characters} ({size})"
msgstr ""

#: cddagl/ui.py:1675
msgid ""
"No executable found in the downloaded archive. You might want to restore "
"your previous version."
msgstr "ダウンロードしたアーカイブの中に実行ファイルがありません。以前のバージョンを復元することをおすすめします。"

#: cddagl/ui.py:1864
msgid "build still in progress!"
msgstr ""

#: cddagl/ui.py:1878
msgid "but build failed for:"
msgstr ""

#: cddagl/ui.py:1889
msgid "No changes, same code as previous build!"
msgstr ""

#: cddagl/ui.py:1990
msgid "Platform:"
msgstr "プラットフォーム:"

#: cddagl/ui.py:1991
msgid "Windows x64 (64-bit)"
msgstr ""

#: cddagl/ui.py:1992
msgid "Windows x86 (32-bit)"
msgstr ""

#: cddagl/ui.py:1993
msgid "Available builds:"
msgstr "有効なビルド:"

#: cddagl/ui.py:1994
msgid "Refresh"
msgstr "更新"

#: cddagl/ui.py:1995
msgid "Changelog"
msgstr "更新履歴"

#: cddagl/ui.py:1997 cddagl/ui.py:3993
msgid "Update/Installation"
msgstr "アップデート/インストール"

#: cddagl/ui.py:2030
msgid "Cannot update or install the game since no build was found"
msgstr ""

#: cddagl/ui.py:2056
msgid "Cannot install the game if the game directory is not empty"
msgstr ""

#: cddagl/ui.py:2073
msgid "Game is up to date"
msgstr "ゲームは最新版です"

#: cddagl/ui.py:2074
msgid "You already have the latest version."
msgstr "すでに最新版がインストールされています。"

#: cddagl/ui.py:2076
msgid "Are you sure you want to update your game?"
msgstr "アップデートを開始しますか？"

#: cddagl/ui.py:2078
msgid "Update the game again"
msgstr "もう一度アップデートする"

#: cddagl/ui.py:2080
msgid "I do not need to update the game again"
msgstr "もう一度アップデートしない"

#: cddagl/ui.py:2108
msgid "Cannot install game on a file"
msgstr "ファイルへのインストールが出来ません"

#: cddagl/ui.py:2148 cddagl/ui.py:2170 cddagl/ui.py:2200 cddagl/ui.py:2225
#: cddagl/ui.py:2248
msgid "Update cancelled"
msgstr "アップデートが中止されました"

#: cddagl/ui.py:2151 cddagl/ui.py:2173 cddagl/ui.py:2203 cddagl/ui.py:2228
#: cddagl/ui.py:2251
msgid "Installation cancelled"
msgstr "インストールが中止されました"

#: cddagl/ui.py:2353 cddagl/ui.py:2418 cddagl/ui.py:4679 cddagl/ui.py:4874
#: cddagl/ui.py:7121 cddagl/ui.py:7308
msgid "Downloading: {0}"
msgstr "ダウンロード中: {0}"

#: cddagl/ui.py:2389 cddagl/ui.py:4114
msgid "Cancel update"
msgstr "アップデート中止"

#: cddagl/ui.py:2391
msgid "Cancel installation"
msgstr "インストール中止"

#: cddagl/ui.py:2457 cddagl/ui.py:4747 cddagl/ui.py:4910 cddagl/ui.py:7193
#: cddagl/ui.py:7356
msgid "Testing downloaded file archive"
msgstr "ダウンロード済みのアーカイブをテスト中"

#: cddagl/ui.py:2494 cddagl/ui.py:4794 cddagl/ui.py:4956 cddagl/ui.py:7230
#: cddagl/ui.py:7401
msgid "Downloaded archive is invalid"
msgstr "ダウンロード済みのアーカイブが無効です"

#: cddagl/ui.py:2504
msgid "Could not download game"
msgstr "ダウンロード出来ませんでした"

#: cddagl/ui.py:2532
#, fuzzy
msgid "Deleting {name}"
msgstr "解析中 {name}"

#: cddagl/ui.py:2533 cddagl/ui.py:2539 cddagl/ui.py:3115
msgid "previous_version directory"
msgstr "以前のバージョンのディレクトリ"

#: cddagl/ui.py:2538
msgid "Update cancelled - Could not delete the {name}."
msgstr ""

#: cddagl/ui.py:2577
msgid "Backing up current game"
msgstr "現在のゲームをバックアップ中"

#: cddagl/ui.py:2621
msgid "Backing up {0}"
msgstr "バックアップ中 {0}"

#: cddagl/ui.py:2640
msgid "Could not move {srcpath} in {dstpath} ."
msgstr ""

#: cddagl/ui.py:2723 cddagl/ui.py:5089 cddagl/ui.py:7558
msgid "Extracting {0}"
msgstr "展開中 {0}"

#: cddagl/ui.py:2733
msgid "Cannot extract game archive"
msgstr "展開に失敗しました"

#: cddagl/ui.py:2735
msgid ""
"\n"
"<p>The launcher failed to extract the game archive.</p>\n"
"<p>It received the following error from the operating system: {error}</p>"
msgstr ""
"\n"
"<p>アーカイブの展開に失敗しました。</p>\n"
"<p>システムから以下のエラーを受け取りました。: {error}</p>"

#: cddagl/ui.py:2812
msgid "{0} directory"
msgstr "{0} ディレクトリ"

#: cddagl/ui.py:2870
msgid "Restoring custom tilesets"
msgstr "カスタムタイルセットを復元中"

#: cddagl/ui.py:2913
msgid "Restoring custom soundpacks"
msgstr "カスタムサウンドパックを復元中"

#: cddagl/ui.py:2962
msgid "{name} soundpack"
msgstr "{name} サウンドパック"

#: cddagl/ui.py:2992
msgid "Restoring custom mods"
msgstr "カスタムModを復元中"

#: cddagl/ui.py:3027
#, fuzzy
msgid "Restoring user custom mods"
msgstr "カスタムModを復元中"

#: cddagl/ui.py:3067
msgid "Restoring user-default-mods.json"
msgstr "ユーザーデフォルトMod設定を復元中"

#: cddagl/ui.py:3081
msgid "Restoring custom fonts"
msgstr "カスタムフォントを復元中"

#: cddagl/ui.py:3136
msgid "Update completed"
msgstr "アップデート完了"

#: cddagl/ui.py:3138
msgid "Installation completed"
msgstr "インストール完了"

#: cddagl/ui.py:3193 cddagl/ui.py:4201 cddagl/ui.py:4997 cddagl/ui.py:5977
#: cddagl/ui.py:6056 cddagl/ui.py:6408 cddagl/ui.py:6494 cddagl/ui.py:7465
#: cddagl/ui.py:8476 cddagl/ui.py:8545
msgid "{bytes_read}/{total_bytes}"
msgstr ""

#: cddagl/ui.py:3202 cddagl/ui.py:4210 cddagl/ui.py:5006 cddagl/ui.py:5970
#: cddagl/ui.py:6066 cddagl/ui.py:6400 cddagl/ui.py:6504 cddagl/ui.py:7475
#: cddagl/ui.py:8469 cddagl/ui.py:8555
msgid "{bytes_sec}/s"
msgstr ""

#: cddagl/ui.py:3220
msgid "Fetching remote builds"
msgstr "リモートビルド取り込み中"

#: cddagl/ui.py:3226 cddagl/ui.py:3264
msgid "Fetching: {url}"
msgstr "取り込み中: {url}"

#: cddagl/ui.py:3323
msgid ""
"You have {remaining} request(s) remaining for accessing GitHub API.\n"
"You will have to wait until {datetime} to get more requests.\n"
"Those requests are needed to get the available builds.\n"
"If you keep running low on those remaining requests, avoid quickly "
"refreshing often\n"
" for theavailable builds. For more information, search GitHub API rate "
"limiting."
msgstr ""

#: cddagl/ui.py:3340
msgid ""
"Could not find remote builds [HTTP {status_code}] ({reason}) when "
"requesting {url}"
msgstr ""

#: cddagl/ui.py:3419
msgid "{number} ({delta})"
msgstr ""

#: cddagl/ui.py:3429
#, fuzzy
msgid " - build unavailable"
msgstr "バックアップ使用可能"

#: cddagl/ui.py:3434
msgid " - latest build available"
msgstr ""

#: cddagl/ui.py:3468
msgid "Could not find remote builds"
msgstr "リモートビルドがありません"

#: cddagl/ui.py:3501
msgid "<h3>Loading changelog...</h3>"
msgstr "<h3>更新履歴を読み込み中...</h3>"

#: cddagl/ui.py:3506
#, fuzzy
msgid "Fetching latest build changelogs"
msgstr "リモートビルド取り込み中"

#: cddagl/ui.py:3549
msgid "<h3>Parsing changelog...</h3>"
msgstr "<h3>更新履歴を解析中...</h3>"

#: cddagl/ui.py:3608
msgid "About CDDA Game Launcher"
msgstr "About CDDA Game について"

#: cddagl/ui.py:3610
msgid ""
"\n"
"<p>CDDA Game Launcher version {version}</p>\n"
"\n"
"<p>Get the latest release <a\n"
"href=\"https://github.com/remyroy/CDDA-Game-Launcher/releases\">on "
"GitHub</a>.</p>\n"
"\n"
"<p>Please report any issue <a\n"
"href=\"https://github.com/remyroy/CDDA-Game-Launcher/issues/new\">on "
"GitHub</a>.\n"
"</p>\n"
"\n"
"<p>If you like the CDDA Game Launcher, you can buy me a beer by donating\n"
"bitcoins to <a href=\"bitcoin:3N2BRM61bZLuFRHjSj2Lhtw6DrwPUGeTvV\">\n"
"3N2BRM61bZLuFRHjSj2Lhtw6DrwPUGeTvV</a> <img src=\"btc-qr.png\"> or by "
"donating\n"
"ethers to <a "
"href=\"https://etherscan.io/address/0xdb731476e913d75061a78105c3d1b5a7a03aa21b\">"
"\n"
"0xDb731476e913d75061A78105C3D1b5A7a03Aa21B</a>\n"
"<img src=\"eth-qr.png\">.</p>\n"
"\n"
"<p>Thanks to the following people for their efforts in translating the "
"CDDA Game\n"
"Launcher</p>\n"
"<ul>\n"
"<li>Russian: Daniel from <a "
"href=\"http://cataclysmdda.ru/\">cataclysmdda.ru</a>\n"
"and Night_Pryanik</li>\n"
"<li>Italian: Rettiliano Verace from <a\n"
"href=\"http://emigrantebestemmiante.blogspot.com\">Emigrante "
"Bestemmiante</a></li>\n"
"<li>French: Rémy Roy</li>\n"
"</ul>\n"
"\n"
"<p>Thanks to <a href=\"http://mattahan.deviantart.com/\">Paul Davey aka\n"
"Mattahan</a> for the permission to use his artwork for the launcher "
"icon.</p>\n"
"\n"
"<p>Copyright (c) 2015-2019 Rémy Roy</p>\n"
"\n"
"<p>Permission is hereby granted, free of charge, to any person obtaining "
"a copy\n"
"of this software and associated documentation files (the \"Software\"), "
"to deal\n"
"in the Software without restriction, including without limitation the "
"rights\n"
"to use, copy, modify, merge, publish, distribute, sublicense, and/or sell"
"\n"
"copies of the Software, and to permit persons to whom the Software is\n"
"furnished to do so, subject to the following conditions:</p>\n"
"\n"
"<p>The above copyright notice and this permission notice shall be "
"included in\n"
"all copies or substantial portions of the Software.</p>\n"
"\n"
"<p>THE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND, "
"EXPRESS OR\n"
"IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\n"
"FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL "
"THE\n"
"AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\n"
"LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING "
"FROM,\n"
"OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN"
" THE\n"
"SOFTWARE.</p>\n"
"\n"
msgstr ""
"\n"
"<p>CDDA Game Launcher version {version}</p>\n"
"\n"
"<p>最新版はこちら<a\n"
"href=\"https://github.com/remyroy/CDDA-Game-Launcher/releases\">on "
"GitHub</a>.</p>\n"
"\n"
"<p>ご意見をお寄せ下さい<a\n"
"href=\"https://github.com/remyroy/CDDA-Game-Launcher/issues/new\">on "
"GitHub</a>.\n"
"</p>\n"
"\n"
"<p>CDDA Game Launcherを気に入りましたか？寄付をして開発者にビールをおごれます\n"
"ビットコインはこちら <a href=\"bitcoin:3N2BRM61bZLuFRHjSj2Lhtw6DrwPUGeTvV\">\n"
"3N2BRM61bZLuFRHjSj2Lhtw6DrwPUGeTvV</a> <img src=\"btc-qr.png\"> \n"
"ethersはこちら <a "
"href=\"https://etherscan.io/address/0xdb731476e913d75061a78105c3d1b5a7a03aa21b\">"
"\n"
"0xDb731476e913d75061A78105C3D1b5A7a03Aa21B</a>\n"
"<img src=\"eth-qr.png\">.</p>\n"
"\n"
"<p>CDDA Game Launcherの翻訳に尽力してくれている皆様に感謝します。</p>\n"
"<ul>\n"
"<li>Russian: Daniel from <a "
"href=\"http://cataclysmdda.ru/\">cataclysmdda.ru</a>\n"
"and Night_Pryanik</li>\n"
"<li>Italian: Rettiliano Verace from <a\n"
"href=\"http://emigrantebestemmiante.blogspot.com\">Emigrante "
"Bestemmiante</a></li>\n"
"<li>French: Rémy Roy</li>\n"
"</ul>\n"
"\n"
"<p>アイコンを提供してくれた <a href=\"http://mattahan.deviantart.com/\">Paul Davey "
"aka\n"
"Mattahan</a> に感謝します。</p>\n"
"\n"
"<p>Copyright (c) 2015-2019 Rémy Roy</p>\n"
"\n"
"<p>Permission is hereby granted, free of charge, to any person obtaining "
"a copy\n"
"of this software and associated documentation files (the \"Software\"), "
"to deal\n"
"in the Software without restriction, including without limitation the "
"rights\n"
"to use, copy, modify, merge, publish, distribute, sublicense, and/or sell"
"\n"
"copies of the Software, and to permit persons to whom the Software is\n"
"furnished to do so, subject to the following conditions:</p>\n"
"\n"
"<p>The above copyright notice and this permission notice shall be "
"included in\n"
"all copies or substantial portions of the Software.</p>\n"
"\n"
"<p>THE SOFTWARE IS PROVIDED \"AS IS\", WITHOUT WARRANTY OF ANY KIND, "
"EXPRESS OR\n"
"IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\n"
"FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL "
"THE\n"
"AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\n"
"LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING "
"FROM,\n"
"OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN"
" THE\n"
"SOFTWARE.</p>\n"
"\n"

#: cddagl/ui.py:3702 cddagl/ui.py:3769
msgid "System language or best match ({locale})"
msgstr "システム言語または推奨 ({locale})"

#: cddagl/ui.py:3712
msgid "{locale_name} - {english_name}"
msgstr ""

#: cddagl/ui.py:3764
msgid "Command line parameters:"
msgstr "コマンドラインパラメータ:"

#: cddagl/ui.py:3766
msgid "Keep the launcher opened after launching the game"
msgstr "ゲーム起動後にランチャーを閉じない"

#: cddagl/ui.py:3767
msgid "Language:"
msgstr "言語:"

#: cddagl/ui.py:3771
msgid "Allow multiple instances of the launcher to be started"
msgstr "複数起動を許可する"

#: cddagl/ui.py:3774
msgid "Use the launcher directory as the game directory"
msgstr "ランチャーが存在するディレクトリをゲームティレクトリとする"

#: cddagl/ui.py:3778
msgid "Launcher"
msgstr "ランチャー"

#: cddagl/ui.py:3976
msgid "Do not copy or move the save directory"
msgstr "セーブディレクトリをコピー・移動しない"

#: cddagl/ui.py:3978
msgid ""
"If your save directory size is large, it might take a long time to copy "
"it during the update process.\n"
"This option might help you speed the whole thing but your previous "
"version will lack the save directory."
msgstr ""
"セーブディレクトリが巨大になると、アップデートの際それをコピーするために長時間を要します。\n"
"この設定を有効にすることで高速化できますが、以前のバージョンのセーブディレクトリは欠落します。"

#: cddagl/ui.py:3983
msgid "Keep a copy of the downloaded archive in the following directory:"
msgstr "ダウンロードしたアーカイブのコピーを保存するディレクトリ:"

#: cddagl/ui.py:3986
msgid "Automatically refresh builds list every"
msgstr "ビルド一覧の自動更新間隔"

#: cddagl/ui.py:3987
msgid "minutes"
msgstr "分"

#: cddagl/ui.py:3988
msgid "Remove previous version after update (not recommended)"
msgstr "アップデート後に以前のバージョンを削除する(非推奨)"

#: cddagl/ui.py:3990
msgid ""
"Permanently delete files instead of moving them in the recycle bin (not "
"recommended)"
msgstr ""

#: cddagl/ui.py:4045
msgid "Archive directory"
msgstr "アーカイブディレクトリ"

#: cddagl/ui.py:4076
msgid "Progress:"
msgstr "進捗:"

#: cddagl/ui.py:4085 cddagl/ui.py:4526 cddagl/ui.py:5285 cddagl/ui.py:5312
#: cddagl/ui.py:6960 cddagl/ui.py:7795 cddagl/ui.py:7822
msgid "Url:"
msgstr "URL:"

#: cddagl/ui.py:4096 cddagl/ui.py:4530 cddagl/ui.py:6964
msgid "Size:"
msgstr "サイズ:"

#: cddagl/ui.py:4105
msgid "Speed:"
msgstr "速度:"

#: cddagl/ui.py:4124
msgid "CDDA Game Launcher self-update"
msgstr "CDDA Game Launcher セルフアップデート"

#: cddagl/ui.py:4237
msgid ""
"This {name} cannot be directly downloaded by the launcher. You have to "
"use your browser to download it."
msgstr "{name}はランチャーから直接ダウンロード出来ません。ブラウザからダウンロードして下さい。"

#: cddagl/ui.py:4244
msgid "1. Open the URL in your browser."
msgstr "1. URLをブラウザで開く。"

#: cddagl/ui.py:4259
msgid "2. Download the {name} on that page and wait for the download to complete."
msgstr "2. {name}のダウンロードが完了するのを待つ。"

#: cddagl/ui.py:4265
msgid "3. Select the downloaded archive."
msgstr "3. ダウンロードしたアーカイブを選択する。"

#: cddagl/ui.py:4290
msgid "Install this {name}"
msgstr "{name}をインストール"

#: cddagl/ui.py:4296
msgid "Do not install"
msgstr "インストールしない"

#: cddagl/ui.py:4307
msgid "Browser download"
msgstr "ブラウザダウンロード"

#: cddagl/ui.py:4312
msgid "Downloaded archive"
msgstr "アーカイブをダウンロードしました"

#: cddagl/ui.py:4313
msgid "Archive files {formats}"
msgstr "アーカイブファイル {formats}"

#: cddagl/ui.py:4322
#, fuzzy
msgid "File not found"
msgstr "実行ファイルがありません"

#: cddagl/ui.py:4324
msgid ""
"{filepath} is not an existing file on your system. Make sure to download "
"the archive with your browser. Make sure to select the downloaded archive"
" afterwards."
msgstr ""

#: cddagl/ui.py:4331
msgid "I will try again"
msgstr ""

#: cddagl/ui.py:4503 cddagl/ui.py:6935
msgid "Installed"
msgstr "インストール完了"

#: cddagl/ui.py:4504 cddagl/ui.py:5202 cddagl/ui.py:5258 cddagl/ui.py:6936
#: cddagl/ui.py:7681 cddagl/ui.py:7753
msgid "Disable"
msgstr "無効"

#: cddagl/ui.py:4505 cddagl/ui.py:6937
msgid "Delete"
msgstr "削除"

#: cddagl/ui.py:4506 cddagl/ui.py:6949
msgid "Repository"
msgstr "リポジトリ"

#: cddagl/ui.py:4508
msgid "Add this new soundpack to the repository"
msgstr "この新しいサウンドパックをリポジトリに追加する"

#: cddagl/ui.py:4509
msgid ""
"* Name: [Enter the name of the soundpack]\n"
"* Url: [Enter the Url where we can find the soundpack]\n"
"* Author: [Enter the name of the author]\n"
"* Homepage: [Enter the Url of the author website or where the soundpack "
"was published]\n"
"* Soundpack not found in version: {version}\n"
msgstr ""
"* Name: [サウンドパックの名前]\n"
"* Url: [サウンドパックをダウンロード可能なURL]\n"
"* Author: [作者の名前]\n"
"* Homepage: [作者の公式サイトまたは配布されている場所のURL]\n"
"* Soundpack not found in version: {version}\n"

#: cddagl/ui.py:4517
msgid "<a href=\"{url}\">Suggest a new soundpack on GitHub</a>"
msgstr "<a href=\"{url}\">GitHubで新しいサウンドパックを提案する</a>"

#: cddagl/ui.py:4519 cddagl/ui.py:4978
msgid "Install this soundpack"
msgstr "このサウンドパックをインストールする"

#: cddagl/ui.py:4520 cddagl/ui.py:6951
msgid "Details"
msgstr "詳細"

#: cddagl/ui.py:4521
msgid "View name:"
msgstr "表示名:"

#: cddagl/ui.py:4522 cddagl/ui.py:5673 cddagl/ui.py:6952
msgid "Name:"
msgstr "名前:"

#: cddagl/ui.py:4528 cddagl/ui.py:5252 cddagl/ui.py:6962 cddagl/ui.py:7743
msgid "Path:"
msgstr "パス:"

#: cddagl/ui.py:4531 cddagl/ui.py:6965
msgid "Home page:"
msgstr "ホームページ:"

#: cddagl/ui.py:4628
msgid "Soundpack already present"
msgstr "サウンドパックがすでに存在しています"

#: cddagl/ui.py:4630
msgid ""
"It seems this soundpack is already installed. The launcher will not "
"overwrite the soundpack if it has the same directory name. You might want"
" to delete the soundpack first if you want to update it. Also, there can "
"only be a single soundpack with the same name value available in the "
"game."
msgstr "このサウンドパックはすでにインストールされているようです。同じ名前のディレクトリが存在する場合、本ランチャーはサウンドパックを上書きしません。アップデートする場合はまずサウンドパックを削除してください。同じ名前のサウンドパックはゲーム中で1つしか有効になりません。"

#: cddagl/ui.py:4637
msgid "Are you sure you want to install the {view} soundpack?"
msgstr "サウンドパック {view} をインストールしますか？"

#: cddagl/ui.py:4640
msgid "Install the soundpack"
msgstr "サウンドパックをインストール"

#: cddagl/ui.py:4642 cddagl/ui.py:7082
msgid "Do not install again"
msgstr "もう一度インストールしない"

#: cddagl/ui.py:4714 cddagl/ui.py:4733
msgid "Cancel soundpack installation"
msgstr "サウンドパックのインストールを中止"

#: cddagl/ui.py:4755 cddagl/ui.py:4800 cddagl/ui.py:4918 cddagl/ui.py:4962
#: cddagl/ui.py:7202 cddagl/ui.py:7236 cddagl/ui.py:7364 cddagl/ui.py:7407
msgid "Selected file is a bad archive file"
msgstr "選択されたアーカイブファイルが壊れています"

#: cddagl/ui.py:4762 cddagl/ui.py:4925 cddagl/ui.py:7209 cddagl/ui.py:7371
msgid "Selected file is a password protected archive file"
msgstr "選択されたアーカイブにはパスワードがかかっています"

#: cddagl/ui.py:4782 cddagl/ui.py:4944 cddagl/ui.py:7389
msgid "Unknown downloaded archive format ({extension})"
msgstr "不明な圧縮フォーマットです ({extension})"

#: cddagl/ui.py:4835 cddagl/ui.py:7272
msgid "Soundpack installation cancelled"
msgstr "サウンドパックのインストールを中止しました"

#: cddagl/ui.py:5118
msgid "Finding the soundpack"
msgstr "サウンドパックを探しています"

#: cddagl/ui.py:5146
msgid ""
"Soundpack installation cancelled - There is no soundpack in the "
"downloaded archive"
msgstr "サウンドパックのインストールを中止しました - ダウンロードしたアーカイブの中にサウンドパックが存在しません"

#: cddagl/ui.py:5156
msgid ""
"Soundpack installation cancelled - There is already a {basename} "
"directory in {soundpacks_dir}"
msgstr "サウンドパックのインストールを中止しました - すでに {basename} が {soundpacks_dir} に存在しています"

#: cddagl/ui.py:5162
msgid "Soundpack installation completed"
msgstr "サウンドパックのインストールが完了しました"

#: cddagl/ui.py:5186 cddagl/ui.py:5404 cddagl/ui.py:7664 cddagl/ui.py:7918
msgid " (Disabled)"
msgstr "(無効)"

#: cddagl/ui.py:5187 cddagl/ui.py:5260 cddagl/ui.py:7665 cddagl/ui.py:7755
msgid "Enable"
msgstr "有効"

#: cddagl/ui.py:5218
msgid "Delete soundpack"
msgstr "サウンドパックを削除"

#: cddagl/ui.py:5219
msgid "This will delete the soundpack directory. It cannot be undone."
msgstr "サウンドパックをディレクトリごと削除します。この操作は元に戻せません。"

#: cddagl/ui.py:5221
msgid "Are you sure you want to delete the {view} soundpack?"
msgstr "サウンドパック {view} を削除してよろしいですか?"

#: cddagl/ui.py:5223
msgid "Delete the soundpack"
msgstr "サウンドパックを削除"

#: cddagl/ui.py:5225
msgid "I want to keep the soundpack"
msgstr "サウンドパックを削除しない"

#: cddagl/ui.py:5234
msgid "Soundpack deletion cancelled"
msgstr "サウンドパックの削除を中止しました"

#: cddagl/ui.py:5239
msgid "Soundpack deleted"
msgstr "サウンドパックを削除しました"

#: cddagl/ui.py:5300 cddagl/ui.py:7810
msgid "Getting remote size"
msgstr "リモートサイズ取得中"

#: cddagl/ui.py:5660
msgid "Backups available"
msgstr "バックアップ使用可能"

#: cddagl/ui.py:5661
msgid "Manual backup"
msgstr "手動バックアップ"

#: cddagl/ui.py:5662
msgid "Automatic backups"
msgstr "自動バックアップ"

#: cddagl/ui.py:5664 cddagl/ui.py:5788 cddagl/ui.py:5817 cddagl/ui.py:6104
msgid "Restore backup"
msgstr "バックアップを復元"

#: cddagl/ui.py:5665
msgid "Refresh list"
msgstr "一覧を更新"

#: cddagl/ui.py:5666 cddagl/ui.py:6124
msgid "Delete backup"
msgstr "バックアップを削除"

#: cddagl/ui.py:5667
msgid "Do not backup the current saves before restoring a backup"
msgstr "バックアップ復元前に現在のセーブをバックアップしない"

#: cddagl/ui.py:5669
msgid "Name"
msgstr "名前"

#: cddagl/ui.py:5670
msgid "Modified"
msgstr "変更"

#: cddagl/ui.py:5670
msgid "Worlds"
msgstr "ワールド"

#: cddagl/ui.py:5670
msgid "Characters"
msgstr "キャラクター"

#: cddagl/ui.py:5670
msgid "Actual size"
msgstr "無圧縮サイズ"

#: cddagl/ui.py:5671
msgid "Compressed size"
msgstr "圧縮後サイズ"

#: cddagl/ui.py:5671
msgid "Compression ratio"
msgstr "圧縮率"

#: cddagl/ui.py:5671
msgid "Modified date"
msgstr "変更日"

#: cddagl/ui.py:5674 cddagl/ui.py:6542
msgid "Backup current saves"
msgstr "現在のセーブをバックアップ"

#: cddagl/ui.py:5676
msgid "Backup saves before game launch"
msgstr "ゲーム起動前にセーブをバックアップ"

#: cddagl/ui.py:5677
msgid "Backup saves after game end"
msgstr "ゲーム終了後にセーブをバックアップ"

#: cddagl/ui.py:5679
msgid ""
"This option will only work if you also have the option to keep the "
"launcher opened after launching the game in the settings tab."
msgstr "この設定を使用するには、設定タブの「ゲーム起動後にランチャーを閉じない」を有効にして下さい。"

#: cddagl/ui.py:5683
msgid "Maximum automatic backups count:"
msgstr "自動バックアップ作成数上限:"

#: cddagl/ui.py:5786 cddagl/ui.py:5815 cddagl/ui.py:5847
msgid "Restore backup cancelled"
msgstr "バックアップの復元が中止されました"

#: cddagl/ui.py:5871
msgid "before_last_restore"
msgstr ""

#: cddagl/ui.py:5913 cddagl/ui.py:6004
msgid "Cancel restore backup"
msgstr "バックアップの復元を中止"

#: cddagl/ui.py:5945
msgid "Could not rename the save directory"
msgstr "セーブディレクトリをリネーム出来ませんでした"

#: cddagl/ui.py:5950
msgid "Could not remove the save file"
msgstr "セーブファイルを削除出来ませんでした"

#: cddagl/ui.py:5965
msgid "Extracting backup"
msgstr "バックアップ展開中"

#: cddagl/ui.py:6027
msgid "Extracting {filename}"
msgstr "展開中 {filename}"

#: cddagl/ui.py:6048
msgid "{backup_name} backup restored"
msgstr "{backup_name} バックアップを復元しました"

#: cddagl/ui.py:6125
msgid "This will delete the backup file. It cannot be undone."
msgstr "バックアップファイルを削除します。この操作は元に戻せません。"

#: cddagl/ui.py:6127
msgid "Are you sure you want to delete the <strong>{filename}</strong> backup?"
msgstr "<strong>{filename}</strong> のバックアップを削除してよろしいですか？"

#: cddagl/ui.py:6130
msgid "Delete the backup"
msgstr "バックアップを削除"

#: cddagl/ui.py:6132
msgid "I want to keep the backup"
msgstr "バックアップを削除しない"

#: cddagl/ui.py:6141
msgid "Backup deletion cancelled"
msgstr "バックアップ削除を中止しました"

#: cddagl/ui.py:6146
msgid "Backup deleted"
msgstr "バックアップを削除しました"

#: cddagl/ui.py:6161 cddagl/ui.py:6203
msgid "Manual backup cancelled"
msgstr "手動バックアップを中止しました"

#: cddagl/ui.py:6209
msgid "manual_backup"
msgstr ""

#: cddagl/ui.py:6256
msgid "Save directory not found"
msgstr "セーブディレクトリが見つかりません"

#: cddagl/ui.py:6275
msgid "Could not delete previous backup archive"
msgstr "以前のバックアップアーカイブを削除出来ませんでした"

#: cddagl/ui.py:6364
msgid "Cancel backup"
msgstr "バックアップを中止"

#: cddagl/ui.py:6367
msgid "Searching for save files"
msgstr "セーブファイル検索中"

#: cddagl/ui.py:6378
msgid "Found {filename} in {path}"
msgstr "場所 {path} で {filename} を発見しました"

#: cddagl/ui.py:6397
msgid "Compressing save files"
msgstr "セーブファイル圧縮中"

#: cddagl/ui.py:6463
msgid "Compressing {filename}"
msgstr "圧縮中 {filename}"

#: cddagl/ui.py:6485
msgid "Saves backup completed"
msgstr "セーブのバックアップが完了しました"

#: cddagl/ui.py:6939
msgid "Add this new mod to the repository"
msgstr "この新しいModをリポジトリに追加する"

#: cddagl/ui.py:6940
msgid ""
"* Name: [Enter the name of the mod]\n"
"* Url: [Enter the Url where we can find the mod]\n"
"* Author: [Enter the name of the author]\n"
"* Homepage: [Enter the Url of the author website or where the mod was "
"published]\n"
"* Mod not found in version: {version}\n"
msgstr ""
"* Name: [Modの名前]\n"
"* Url: [Modをダウンロード可能なURL]\n"
"* Author: [作者の名前]\n"
"* Homepage: [作者の公式サイトまたは配布されている場所のURL]\n"
"* Mod not found in version: {version}\n"

#: cddagl/ui.py:6947
msgid "<a href=\"{url}\">Suggest a new mod on GitHub</a>"
msgstr "<a href=\"{url}\">GitHubで新しいModを提案する</a>"

#: cddagl/ui.py:6950 cddagl/ui.py:7423
msgid "Install this mod"
msgstr "このModをインストールする"

#: cddagl/ui.py:6953
msgid "Ident:"
msgstr "ID:"

#: cddagl/ui.py:6954
msgid "Author:"
msgstr "作者:"

#: cddagl/ui.py:6955
msgid "Description:"
msgstr "説明:"

#: cddagl/ui.py:6956
msgid "Category:"
msgstr "分類:"

#: cddagl/ui.py:7069
msgid "Mod already present"
msgstr "Modがすでに存在しています"

#: cddagl/ui.py:7070
msgid ""
"It seems this mod is already installed. The launcher will not overwrite "
"the mod if it has the same directory name. You might want to delete the "
"mod first if you want to update it. Also, there can only be a single mod "
"with the same ident value available in the game."
msgstr "このModはすでにインストールされているようです。同じ名前のディレクトリが存在する場合、本ランチャーはModを上書きしません。アップデートする場合はまずModを削除してください。同じ名前のModはゲーム中で1つしか有効になりません。"

#: cddagl/ui.py:7077
msgid "Are you sure you want to install the {name} mod?"
msgstr "Mod {name} をインストールしてよろしいですか？"

#: cddagl/ui.py:7080
msgid "Install the mod"
msgstr "Modをインストール"

#: cddagl/ui.py:7156 cddagl/ui.py:7182
msgid "Cancel mod installation"
msgstr "Modのインストールを中止"

#: cddagl/ui.py:7176
msgid "Could not find downloaded file archive"
msgstr "ダウンロードしたアーカイブが見つかりませんでした"

#: cddagl/ui.py:7349
msgid "Could not find downloaded archive ({file})"
msgstr "ダウンロードしたアーカイブが見つかりませんでした ({file})"

#: cddagl/ui.py:7587
msgid "Finding the mod(s)"
msgstr "Mod検索中"

#: cddagl/ui.py:7618
msgid "Mod installation cancelled - There is no mod in the downloaded archive"
msgstr "Modのインストールを中止しました - ダウンロードしたアーカイブの中にModが存在しません"

#: cddagl/ui.py:7630
msgid ""
"Mod installation cancelled - There is already a {basename} directory in "
"{mods_dir}"
msgstr "Modのインストールを中止しました - すでに {basename} が {mods_dir} に存在しています。"

#: cddagl/ui.py:7639
msgid "Mod installation completed"
msgstr "Modのインストールが完了しました"

#: cddagl/ui.py:7663 cddagl/ui.py:7680 cddagl/ui.py:7920
msgid "*Error*"
msgstr "*エラー*"

#: cddagl/ui.py:7697
msgid "Delete mod"
msgstr "Modを削除"

#: cddagl/ui.py:7698
msgid "This will delete the mod directory. It cannot be undone."
msgstr "Modディレクトリを削除します。この操作は元に戻せません。"

#: cddagl/ui.py:7700
msgid "Are you sure you want to delete the {view} mod?"
msgstr "Mod {view} を削除してよろしいですか？"

#: cddagl/ui.py:7702
msgid "Delete the mod"
msgstr "Modを削除"

#: cddagl/ui.py:7704
msgid "I want to keep the mod"
msgstr "Modを削除しない"

#: cddagl/ui.py:7713
msgid "Mod deletion cancelled"
msgstr "Modの削除を中止しました"

#: cddagl/ui.py:7718
msgid "Mod deleted"
msgstr "Modを削除しました"

#: cddagl/ui.py:8168 cddagl/ui.py:8404
#, python-format
msgid "Source path '%s' is not a directory"
msgstr "ソースの場所 '%s' はディレクトリではありません"

#: cddagl/ui.py:8199 cddagl/ui.py:8447
msgid "file"
msgid_plural "files"
msgstr[0] "ファイル"

#: cddagl/ui.py:8201
msgid "Analysing {name} - Found {file_count} {files}"
msgstr "解析中 {name} - 検出 {file_count} {files}"

#: cddagl/ui.py:8246 cddagl/ui.py:8310
msgid "Cannot remove directory"
msgstr "ディレクトリを削除できません"

#: cddagl/ui.py:8253 cddagl/ui.py:8316
msgid ""
"\n"
"<p>The launcher failed to remove the following directory: {directory}</p>"
"\n"
"<p>When trying to remove or access {filename}, the launcher raised the\n"
"following error: {error}</p>"
msgstr ""
"\n"
"<p>以下のディレクトリの削除に失敗しました: {directory}</p>\n"
"<p>{filename}へのアクセスを試みましたがエラーが発生しました\n"
": {error}</p>"

#: cddagl/ui.py:8262 cddagl/ui.py:8325
msgid ""
"\n"
"<p>No process seems to be using that file or directory.</p>"
msgstr ""
"\n"
"<p>選択されたファイルまたはディレクトリを使用中のプロセスはありません。</p>"

#: cddagl/ui.py:8265 cddagl/ui.py:8328
msgid ""
"\n"
"<p>The process <strong>{image_file_name} ({pid})</strong> is currently "
"using\n"
"that file or directory. You might need to end it if you want to retry.</p>"
msgstr ""
"\n"
"<p>プロセス <strong>{image_file_name} ({pid})</strong> が、指定された\n"
"ファイルまたはディレクトリを使用中です。 上記プロセス終了後に再試行して下さい。</p>"

#: cddagl/ui.py:8273 cddagl/ui.py:8335
msgid "Do you want to retry removing this directory?"
msgstr "ディレクトリの削除を再試行しますか？"

#: cddagl/ui.py:8276 cddagl/ui.py:8338
msgid "Retry removing the directory"
msgstr "ディレクトリの削除を再試行する"

#: cddagl/ui.py:8357
msgid "Deleting {name} - {entry}"
msgstr "削除中 {name} - {entry}"

#: cddagl/ui.py:8367 cddagl/ui.py:8576
msgid "Analysing {name}"
msgstr "解析中 {name}"

#: cddagl/ui.py:8406
#, python-format
msgid "Destination path '%s' already exists"
msgstr "実行パス '%s' はすでに存在します"

#: cddagl/ui.py:8450
msgid "Analysing {name} - Found {file_count} {files} ({size})"
msgstr "解析中 {name} - 検出 {file_count} {files} ({size})"

#: cddagl/ui.py:8566
msgid "Copying {name} - {entry}"
msgstr "コピー中 {name} - {entry}"

#: cddagl/ui.py:8623
msgid ""
"The CDDA Game Launcher just crashed. An unhandled exception was raised. "
"Here are the details."
msgstr "不明な例外処理が発生し、The CDDA Game Launcher は動作を停止しました。詳細はこちら。"

#: cddagl/ui.py:8635
msgid ""
"\n"
"<p>CDDA Game Launcher version: {version}</p>\n"
"<p>OS: {os} ({bitness})</p>\n"
"<p>Type: {extype}</p>\n"
"<p>Value: {value}</p>\n"
"<p>Traceback:</p>\n"
"<code>{traceback}</code>\n"
msgstr ""

#: cddagl/ui.py:8650
msgid "Unhandled exception: [Enter a title]"
msgstr ""

#: cddagl/ui.py:8651
msgid ""
"* Description: [Enter what you did and what happened]\n"
"* Version: {version}\n"
"* OS: {os} ({bitness})\n"
"* Type: `{extype}`\n"
"* Value: {value}\n"
"* Traceback:\n"
"```\n"
"{traceback}\n"
"```\n"
msgstr ""

#: cddagl/ui.py:8666
msgid ""
"Please help us make a better launcher <a href=\"{url}\">by reporting this"
" issue on GitHub</a>."
msgstr ""

#: cddagl/ui.py:8673
msgid "Exit"
msgstr "終了"

#: cddagl/ui.py:8679
msgid "Something went wrong"
msgstr "何かが間違っています"

#: cddagl/ui.py:8693
msgid "Could not find translations for {locale} in {locale_dir} ({info})"
msgstr "{locale} のための翻訳を {locale_dir} 内に発見出来ませんでした({info})"

#~ msgid ""
#~ "\n"
#~ "<p>The launcher failed to remove the "
#~ "following directory: {directory}</p>\n"
#~ "<p>When trying to remove or access "
#~ "{filename}, the launcher raised the\n"
#~ "following error: {error}</p>\n"
#~ msgstr ""
#~ "\n"
#~ "<p>以下のディレクトリの削除に失敗しました: {directory}</p>\n"
#~ "<p>{filename}へのアクセスを試みましたがエラーが発生しました\n"
#~ ": {error}</p>\n"

#~ msgid ""
#~ "\n"
#~ "<p>No process seems to be using that file or directory.</p>\n"
#~ msgstr ""
#~ "\n"
#~ "<p>指定されたファイルまたはディレクトリを使用中のプロセスはありません。</p>\n"

#~ msgid ""
#~ "\n"
#~ "<p>The process <strong>{image_file_name} "
#~ "({pid})</strong> is currently using\n"
#~ "that file or directory. You might "
#~ "need to end it if you want "
#~ "to retry.</p>\n"
#~ msgstr ""
#~ "\n"
#~ "<p>プロセス <strong>{image_file_name} ({pid})</strong> が、指定された\n"
#~ "ファイルまたはディレクトリを使用中です。 上記プロセス終了後に再試行して下さい。</p>\n"

#~ msgid "Cannot delete file"
#~ msgstr "ファイルを削除できません"

#~ msgid ""
#~ "\n"
#~ "<p>The launcher failed to delete the following file: {path}</p>\n"
#~ "<p>When trying to remove or access "
#~ "{filename}, the launcher raised the\n"
#~ "following error: {error}</p>\n"
#~ msgstr ""
#~ "\n"
#~ "<p>以下のファイルの削除に失敗しました: {path}</p>\n"
#~ "<p>{filename},へのアクセスを試みましたがエラーが発生しました\n"
#~ ": {error}</p>\n"

#~ msgid "Do you want to retry deleting this file?"
#~ msgstr "ファイルの削除を再試行しますか？"

#~ msgid "Retry deleting the file"
#~ msgstr "ファイルの削除を再試行"

#~ msgid "Graphics:"
#~ msgstr "グラフィック:"

#~ msgid "Tiles"
#~ msgstr "タイル"

#~ msgid "Console"
#~ msgstr "コンソール"

#~ msgid ""
#~ "Update cancelled - Your previous_version "
#~ "directory is most likely unusable now."
#~ " Restoring your previous version will "
#~ "most likely give you a broken "
#~ "game."
#~ msgstr ""
#~ "アップデートが中止されました - "
#~ "以前のバージョンのディレクトリが利用できません。以前のバージョンを復元した場合、正常に動作しないと思われます。"

#~ msgid "{number} ({delta}) - latest"
#~ msgstr "{number} ({delta}) - 最新版"

#~ msgid "Could not download soundpack"
#~ msgstr "サウンドパックをダウンロード出来ませんでした"

